docs/components/Colors.js

```diff
@@ -24,15 +30,10 @@ function Wrapper({ name, theme, colorsObject }) {
                 w={60}
               />
               <Box px="sm">
-                <Text
-                  variant="h6"
-                  as="span"
-                  textTransform="capitalize"
-                  color={isLight && 'light.900'}
-                >
+                <Text variant="h6" as="span" textTransform="capitalize" color={textColor}>
```

Comment:
is not the same rule than before

Reply:
In the case of dark mode, we don't pass color for the `Text` component and got an error because we pass "undefined". What we need to use in this another case ?

Reply:
@Geminii `{...(isLight  && { color: 'light.900' })}` could work?

Reply:
This works, thanks @cnairiwttj 🙏